openff/bespokefit/schema/smirnoff.py

```diff
@@ -95,8 +95,8 @@ class VdWHyperparameters(BaseSMIRKSHyperparameters):
 
     type: Literal["vdW"] = "vdW"
 
-    priors: Dict[Literal["epsilon", "rmin_half"], PositiveFloat] = Field(
-        {"epsilon": 0.1, "rmin_half": 0.1}, description=""
```

Comment:
Is there a reason to use `sigma` over `rmin_half` here?

Reply:
This was just to align the `VdWSMIRKS` and `VdWHyperparameters` objects to use the same naming happy to go with either though if you would prefer `rmin_half`?

Reply:
Ah that makes sense! Probably best to keep the two aligned!